wsmaster/codenvy-hosted-api-organization/src/main/java/com/codenvy/organization/spi/OrganizationDistributedResourcesDao.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.organization.spi;
+
+import com.codenvy.organization.spi.impl.OrganizationDistributedResourcesImpl;
+
+import org.eclipse.che.api.core.NotFoundException;
+import org.eclipse.che.api.core.Page;
+import org.eclipse.che.api.core.ServerException;
+
+/**
+ * Defines data access object contract for {@link OrganizationDistributedResourcesImpl}.
+ *
+ * @author Sergii Leschenko
+ */
+public interface OrganizationDistributedResourcesDao {
+    /**
+     * Stores distributed resources for suborganization.
```

Comment:
I guess it will be helpful to describe here how to store resources when there are already exist resources for the organization.